Godeps/LICENSES

```diff
@@ -231,7 +231,7 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-= vendor/bitbucket.org/bertimus9/systemstat/LICENSE 6f7ba8122a9b3146266dafa39c5b8ee3
+= vendor/bitbucket.org/bertimus9/systemstat/LICENSE 
```

Comment:
thanks to @cblecker it's clear that these should not be removed. the changes in this file are wrong.

Reply:
I'm not sure about  quite sure about the standard process of updating a vendor package. For klog, I think these are the suggested cmds: 
>> hack/pin-dependency.sh k8s.io/klog <GIT-COMMIT>
>> hack/update-vendor.sh

I can manually fix the problematic file. But I think the hack scripts may need to be updated as well. Do you know which sig or owner should be aware of the generic deps updates.

Reply:
oh, never mind. I saw the prow test gives the suggested fixes.